### ISSUE TITLE
hammer fails with 'comparison of Symbol with 80 failed'

### ISSUE THREAD

**Issue by lpramuk** [SatelliteQE/robottelo#4311]
- [ ] cli.test_lifecycleenvironment.LifeCycleEnvironmentTestCase.test_positve_list_paths
```
E           CLIReturnCodeError: Command "lifecycle-environment paths" finished with return_code 64
E           stderr contains following message:
E           [ERROR 2017-02-08 12:13:32 Exception] comparison of Symbol with 80 failed
E           comparison of Symbol with 80 failed
E           [ERROR 2017-02-08 12:13:32 Exception] 
E           
E           ArgumentError (comparison of Symbol with 80 failed):
E               /opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli-0.5.1.13/lib/hammer_cli/output/adapter/table.rb:96:in `>='
E               /opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli-0.5.1.13/lib/hammer_cli/output/adapter/table.rb:96:in `block in calculate_column_width'
```
- [ ] cli.test_lifecycleenvironment.LifeCycleEnvironmentTestCase.test_verify_bugzilla_1077386
```
E           CLIReturnCodeError: Command "lifecycle-environment info" finished with return_code 64
E           stderr contains following message:
E           [ERROR 2017-02-08 12:13:44 Exception] comparison of Symbol with 80 failed
E           comparison of Symbol with 80 failed
E           [ERROR 2017-02-08 12:13:44 Exception] 
E           
E           ArgumentError (comparison of Symbol with 80 failed):
E               /opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli-0.5.1.13/lib/hammer_cli/output/adapter/table.rb:96:in `>='
```
- [ ] cli.test_subscription.SubscriptionTestCase.test_positive_manifest_history
```
E           CLIReturnCodeError: Command "subscription manifest-history" finished with return_code 64
E           stderr contains following message:
E           [ERROR 2017-02-08 15:42:34 Exception] comparison of Symbol with 80 failed
E           comparison of Symbol with 80 failed
E           [ERROR 2017-02-08 15:42:34 Exception] 
E           
E           ArgumentError (comparison of Symbol with 80 failed):
```

Most probably this'll be a valid bug. If so we can keep this issue open to have tests decorated with a bz.

**Comment by lpramuk** [comment]
Bug confirmed and reported 
- https://bugzilla.redhat.com/show_bug.cgi?id=1420673 hammer lifecycle-environment commands fail with 'comparison of Symbol with 80 failed'

**Comment by lpramuk** [comment]
Probably no action here. Fix will land with the next compose.

**Comment by rochacbruno** [comment]
Bug Fixed in 6.2.8 3.0

I verified the https://bugzilla.redhat.com/show_bug.cgi?id=1023127 which was blocked by the same BZ.

**Comment by lpramuk** [comment]
VERIFIED https://bugzilla.redhat.com/show_bug.cgi?id=1420673
